scripts/pre-commit.sh

```diff
@@ -11,7 +11,7 @@ if [ -n "$jsfiles" ]; then
     for js in $jsfiles; do
         echo "$js"
         e=$(node_modules/.bin/eslint --quiet --fix $js)
-        if [[ -n "$e" ]]; then
+        if [ -n "$e" ]; then
```

Comment:
wondering if we could also add the check & the extract on the pre-commit ;) that could avoid a failed PR

Reply:
True, added.